libcontainer/integration/checkpoint_test.go

```diff
@@ -136,7 +136,7 @@ func testCheckpoint(t *testing.T, userns bool) {
 	preDumpLog := filepath.Join(preDumpOpts.WorkDirectory, "dump.log")
 
 	if err := container.Checkpoint(preDumpOpts); err != nil {
-		showFile(t, preDumpLog)
+		_ = showFile(t, preDumpLog)
```

Comment:
Since no caller of `showFile()` checks for any errors returned by it, I think it's better to change the function itself to return no error.

Reply:
Updated (not pushed yet)